go-controller/pkg/factory/factory_test.go

```diff
@@ -197,12 +198,13 @@ var _ = Describe("Watch Factory Operations", func() {
 	})
 
 	AfterEach(func() {
-		close(stop)
+		wf.Shutdown(stop)
 	})
 
 	Context("when a processExisting is given", func() {
 		testExisting := func(objType reflect.Type, namespace string, lsel *metav1.LabelSelector) {
-			wf, err := NewWatchFactory(fakeClient, stop)
+			var err error
```

Comment:
you could just define `err` at the top level too...

Reply:
done